versioned/transfer/src/main/java/org/projectnessie/versioned/transfer/ImportResult.java

```diff
@@ -32,9 +33,9 @@ public interface ImportResult {
    * optimization after a repository import.
    *
    * <p>Note: the HEADS of the named references in an export may not match the heads in {@link
-   * HeadsAndForkPoints} when commits happened while the export has been created.
+   * HeadsAndForks} when commits happened while the export has been created.
    */
-  HeadsAndForkPoints headsAndForkPoints();
+  HeadsAndForks headsAndForks();
```

Comment:
Why do we remove `HeadsAndForkPoints` here, but keep it in `CommitLogOptimization`, which is in the same java package as this class?

Reply:
It looks like `CommitLogOptimization` always gets heads and forks from `ImportResult`. I tend to think it might be preferable to change `CommitLogOptimization.headsAndForks` to `HeadsAndForks` and add a utility conversion method to it. WDYT?

Reply:
CLO is only useful for `DatabaseAdapter`, so fine to keep it in CLO

Reply:
ok